buildpipeline/Core-Setup-Linux.json

```diff
@@ -175,12 +181,12 @@
         "SymbolStore"
       ],
       "daysToKeep": 2,
-      "minimumToKeep": 10,
+      "minimumToKeep": 1,
```

Comment:
Is this intentional, or just a difference between them? I would prefer 10, although historically I know the CLI-split defs tend to have short retention policies.